scipy/stats/tests/test_stats.py

```diff
@@ -6828,6 +6827,12 @@ def test_monotonicity(self, variant, method):
         ]
         assert np.all(np.diff(combined_pvalues) >= 0)
 
+    @pytest.mark.parametrize("method", methods)
```

Comment:
Perhaps not necessary, but I think there are other stats functions with different `return` statements for different `method`s. In those cases, we'd want to make sure all of them have been updated.